diagnostic.md

```diff
@@ -150,6 +180,9 @@ div {
 ```
 
 <!-- your answer starts here -->
+The first 3 divs will be in a line from left to right
+the 4th div will begin on the line below
+the 5th will begin below the 4th div
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5